### DISCUSSION TITLE
No changes occured after deployment

### DISCUSSION THREAD

**Discussion by joshypet225** [microsoft/SATechnicalOnboarding#142] [Questions]
I did everything according to the React template but when I checked the Azure link there was no changes made and when I also checked the github page it was blank. Please what can I do

**Comment by AnthonyByansi** [comment]
Hi @joshypet225 
Did you change the configurations for the `package.json` file as well?
![github-pages-deploy](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/2c8e4761-70e6-4033-a62e-a2f7c7b5d06d)

Please remember to also change the following section from build to `dist`
![image](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/30dba276-2f8d-4206-ac06-30d640c1ecc8)
